cli/src/cmd/mod.rs

```diff
@@ -95,6 +95,15 @@ If you are in a subdirectory in a Git repository, try adding `--root .`"#,
         println!("{}", output);
         println!("success.");
     }
+
+    for (name, artifact) in output.clone().into_artifacts() {
+        if let Some(bytecode) = artifact.bytecode {
+            if let Some(bytes) = bytecode.object.as_bytes() {
+                println!("{}: codesize {}", name, bytes.to_vec().len());
+            }
+        }
+    }
```

Comment:
these should prob go into a table view that's gated behind a flag, not always on by default